### PR TITLE
2969

### PR DESCRIPTION
**Pull request by singhmonaaccolite**
2969 - When deleted the file from the file manager it is not getting removed from the disk - Fixed.

### CODE REVIEW

File: common/index.html [hunk 1]
```diff
@@ -176,8 +195,27 @@ <h2>Sample 2: Save to disk (resize on server)</h2>
...
+        })
+        .done (function (data) {
+          console.log ('file was deleted from files manager popup');
+        })
+        .fail (function (err) {
+          console.log ('files manager popup delete problem: ' + JSON.stringify(err));
+        })
+      },
       // Catch video removal from the editor.
-      'froalaEditor.video.removed': function ($vid) {
```
Comment (kapil2704) [inline common/index.html:180]: @singhmonaaccolite why did we remove this ? Was it not working before?

File: django_examples/api/views.py [hunk 2]
```diff
@@ -32,17 +32,20 @@ def upload_files(request):
...
+        with wand.image.Image(filename=filePath) as img:
+            if img.width != img.height:
+                return False
         size = os.path.getsize(filePath)
         if size > 10 * 1024 * 1024:
             return False
         return True
 
     options = {
-        'fieldname': 'myFile',
```
Comment (kapil2704) [inline django_examples/api/views.py:41]: @singhmonaaccolite What is reason to change fieldname ?

File: common/index.html (hunk 1 above)
Comment (singhmonaaccolite) [reply common/index.html:180]: It was not working before as the froalaEditor.video.removed is not an event.

File: django_examples/api/views.py (hunk 2 above)
Comment (singhmonaaccolite) [reply django_examples/api/views.py:41]: We are addressing image here so the field name was changed, as the third editor checks if image height and width is equal or not.

**Pull request merged**